subprojects/fmt.wrap

```diff
@@ -0,0 +1,3 @@
+[wrap-git]
+url = https://github.com/fmtlib/fmt
+revision = HEAD
```

Comment:
Upstream fmt doesn't build with meson. There is a WrapDB port of it, however, so I would recommend:
- delete this file
- run `meson wrap install fmt`